react/features/base/participants/functions.js

```diff
@@ -276,3 +281,44 @@ export function isLocalParticipantModerator(stateful: Object | Function) {
             && (!state['features/base/config'].enableUserRolesBasedOnToken
                 || !state['features/base/jwt'].isGuest));
 }
+
+/**
+ * Returns true if the video of the participant should be rendered.
+ *
+ * @param {Object|Function} stateful - Object or function that can be resolved
+ * to the Redux state.
+ * @param {string} id - The ID of the participant.
+ * @returns {boolean}
+ */
+export function shouldRenderParticipantVideo(
+        stateful: Object | Function, id: string) {
+    const state = toState(stateful);
+    const participant = getParticipantById(state, id);
+
+    if (!participant) {
+        return false;
+    }
+
+    const audioOnly = state['features/base/conference'].audioOnly;
+    const connectionStatus = participant.connectionStatus;
```

Comment:
I'm geting no video for any participant after this change. I suspect it has to do with this line. In the previous code we ` || JitsiParticipantConnectionStatus.ACTIVE,`  it since it can be undefined, so we treat it as active.

Reply:
I did get video, although adding this extra OR makes it quicker to receive video. Thanks for pointing it out